PROTOCOL.md

```diff
@@ -433,6 +433,7 @@ A reference implementation can be found in [the catalyst package of the Apache S
 Type Name | Description
 -|-
 string| UTF-8 encoded string of characters
+long| 8-byte signed integer. Rnage: -9223372036854775808 to 9223372036854775808
```

Comment:
The max value should be `9223372036854775807`.

Reply:
good catch :+1: